.grabl/automation.yml

```diff
@@ -126,6 +126,7 @@ build:
         bazel test //tests/behaviour/graql/language/undefine:checkstyle --test_output=errors
         bazel test //tests/behaviour/graql/language/undefine:test-core --test_output=errors
     test-cluster-failover:
+      machine: 4-core-8-gb
```

Comment:
We were getting intermittent test failures in Grabl because a 2-core machine is not well-suited to running a 3-node Cluster.